### ISSUE TITLE
There is an error in the else statement

### ISSUE THREAD

**Issue by doomfist00** [Zezombye/overpy#138]
```python
globalvar Condition1
globalvar Condition2

@Rule "Test"
@Event global
	if Condition1:
		if Condition2:
			A = A
	else:
		smallMessage(hostPlayer,"Error Occured")

'''
variables {
	global:
		0: A
		1: Condition1
		2: Condition2
}

rule ("Test") {
	event {
		Ongoing - Global;
	}

	actions {
		Skip If(Not(Global Variable(Condition1)), 3);
		Skip If(Not(Global Variable(Condition2)), 2);
		Set Global Variable(A, Global Variable(A));
		Skip(1);
		Small Message(Host Player, Custom String("Error Occured", Null, Null, Null));
	}
}
'''	
```

**Comment by Zezombye** [comment]
Thanks for reporting, I'll take a look at it.

In the meantime you can use the `pass` instruction:

```python
@Rule "Test"
@Event global
	if Condition1:
		if Condition2:
			A = A
		pass
	else:
		smallMessage(hostPlayer,"Error Occured")

**Comment by Zezombye** [comment]
Should be fixed in 4.2.1.
